### PR TITLE
Updating Calendar document

### PR DESCRIPTION
**Pull request by jpwalden**
Added how to heal google sync -chapter, added weekly view and made some other updates as well.

### CODE REVIEW

**Comment by jovirkku** [comment]
If we wanted, line 26 could also tell that the dot colours denote various sub-calendars. The colours can be changed in the "Manage calendars" of the pulley menu.

File: Support/Help_Articles/Calendar/README.md [hunk 1]
```diff
@@ -24,31 +24,43 @@ When your Calendar app opens, you are greeted with the Month view.
...
   <span class="md_figcaption">
   </span>
 </div>
 
-## Day View
-You can access the Day view by first selecting a date in the Month view, and then pressing the selected day at the base of the calendar at the right-hand side ("Today" or the weekday like "Wednesday").
+## Weekly View
+
+* You can access the weekly view from the left top corner by selecting the weekly view (icon in the middle).
+* In the weekly view, it is easier to see the summary of the week's events.
```
Comment (jovirkku) [inline Support/Help_Articles/Calendar/README.md:44]: Even better if you turn the device to the landscape position.

File: Support/Help_Articles/Calendar/README.md [hunk 2]
```diff
@@ -124,10 +136,47 @@ Google calendar and Microsoft Exchange calendar are examples of calendar types,
...
+* You can snooze a calendar alert by flipping the phone (turn it screen downwards) or by pressing the power key shortly.
+
+# How to heal the calendar or contact sync of Google
+This article may turn out helpful in resolving persistent issues in Google sync.
+ 
+It is known that sometimes, even though the actual calendar sync is pulling the appointments from the server to the database in the phone, these appointments do not appear in Sailfish Calendar. Likewise, the contact sync may not do its job.
+The following is a kind of reset of the sync utility. It does not do any harm to the appointments or contacts.
+ 
+## Preparations
+1. You will need the Developer mode and the SSH password. Do as instructed in this help article.
```
Comment (jovirkku) [inline Support/Help_Articles/Calendar/README.md:151]: Link to the document missing?

**Review by jovirkku — CHANGES_REQUESTED** [review]
The modified parts (which are under review) are almost ok. Little needs be reworked.

If we want to improve the unmodified parts (unmodified by this commit) then there is more to do.

**Comment by jpwalden** [comment]
> As I started reading the rich view of the whole doc (not just the changed parts) I noticed the following slight problems - the implementation of the app has changed over the years but the document text not so much...
> 
> If we wanted, line 26 (or line 28) could also tell that the dot colours denote various sub-calendars. The colours can be changed as explained in chapter "Changing calendar colours"
> 
> On the old line 56 or the new line 68, just under "_Creating a new Event in Calendar_", the sentence claims that new events should be created in the Month view. But they can be created in the week and day views, too. All 3 views share the same pulley menu. So, the sentence could be something like: "_When it’s time to mark something important down, pull down “Create new event”_.
> 
> Remove the last part of the sentence, i.e. " _where it shows what you chose for the date or time._" from line 74/86 (end of step 4). Namely, the current OS version has the Accept button in the top right corner. It does not show the selected date or time there.
> 
> Line 77/89: I do not think we have the "read-only" tagging anymore. Do we? If not, the "please note" sentence (2) should be removed. Line 113/125 refers to "read-only", too. I wonder if it is correct. Would Pekka know?

Good observations, following changes made: colours mentioned, fixed the create event part, since that can be indeed done also from weekly and daily view, removed the last part of the sentence in step4. Checked from Pekka that read-only events part is still valid, so didn't remove that.

File: Support/Help_Articles/Calendar/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/Calendar/README.md]: Added comment about landscape

File: Support/Help_Articles/Calendar/README.md [hunk 3]
```diff
@@ -24,31 +24,43 @@ When your Calendar app opens, you are greeted with the Month view.
...
+
+* <a href="Cal-en-3.png" class="narrow-image"><img src="Cal-en-3.png" alt="Weekly view"></a>
+  <span class="md_figcaption">
+  </span>
+</div>
+
+## Day View
 
-At the bottom of the view, you can see events that are coming up if you scroll further down. At the top of the screen, you can see events that are in the past that you can see by scrolling up.
+* You can access the day view from the left top corner by selecting the weekly view (icon in the right).
```
[the remark this answers is not in the document]
Comment (jpwalden) [reply Support/Help_Articles/Calendar/README.md]: Fixed

File: Support/Help_Articles/Calendar/README.md [hunk 4]
```diff
@@ -124,10 +136,47 @@ Google calendar and Microsoft Exchange calendar are examples of calendar types,
...
-There is no week view in the Calendar app.
-Birthday events cannot be created, deleted or modified in Calendar because they are obtained through syncing. It is, however, possible to change or delete a person’s birthday by modifying that person’s information in the People application (phonebook). Thereafter the modification made will appear in Calendar, too.
-The sync schedule for your calendars is the same as for the corresponding email service. For Exchange, there are more refined sync options in Settings > Accounts > Exchange.
-You can snooze a calendar alert by flipping the phone (turn it screen downwards) or by pressing the power key shortly.
+* If you set an end date for your calendar entry, the event will end on the date you’ve specified, but the calendar will treat it as a non-stop event. This means that an event will begin the date you specify, start at the time you specify, and then go on uninterrupted until the end date and time you time you’ve specified.
```
[the remark this answers is not in the document]
Comment (jpwalden) [reply Support/Help_Articles/Calendar/README.md]: That is quite unclear sentence, seems to cause more confusion than clarification, so removed it.

File: Support/Help_Articles/Calendar/README.md (hunk 2 above)
Comment (jpwalden) [reply Support/Help_Articles/Calendar/README.md]: Thanks, that was missing -> added.

**Review by jovirkku — APPROVED** [review]
This looks ok now. 

[at new line 68, step 5, selecting the calendar: I think it is still so that the calendar once selected for an event cannot later be changed anymore - so if one mistakenly selects a wrong calendar then he should open a new event to force it to another calendar]

**Comment by jpwalden** [comment]
> This looks ok now.
> 
> [at new line 68, step 5, selecting the calendar: I think it is still so that the calendar once selected for an event cannot later be changed anymore - so if one mistakenly selects a wrong calendar then he should open a new event to force it to another calendar]

Added a note about that as discussed

**Comment by jovirkku** [comment]
Thanks for the "Added a note about that as discussed"

>

**Comment by jovirkku** [comment]
I was supposed to say that this is a good document now. Ready to be approved.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**